pkg/backporter/backporter.go

```diff
@@ -278,94 +304,218 @@ func GetBugHandler(client bugzilla.Client) HandlerFuncWithErrorReturn {
 	}
 }
 
+func getClonesTemplateData(bugID int, w http.ResponseWriter, client bugzilla.Client, allTargetVersions sets.String) (*ClonesTemplateData, error) {
+	bug, err := client.GetBug(bugID)
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+		wpErr := writePage(w, "Not Found", errorTemplate, fmt.Sprintf("Bug#%d not found", bugID))
+		if wpErr != nil {
+			http.Error(w, "Error building page.", http.StatusInternalServerError)
+			return nil, wpErr
+		}
+		return nil, err
+	}
+	prs, err := client.GetExternalBugPRsOnBug(bug.ID)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wpErr := writePage(w, "Error.", errorTemplate, fmt.Sprintf("Bug#%d - error occured while retreiving list of PRs : %v", bug.ID, err))
+		if wpErr != nil {
+			http.Error(w, "Error building page.", http.StatusInternalServerError)
+			return nil, wpErr
+		}
+		return nil, err
+	}
+	parent, err := client.GetRootForClone(bug)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wpErr := writePage(w, "Error.", errorTemplate, fmt.Sprintf("Bug#%d Details of parent could not be retrieved : %v", bug.ID, err))
+		if wpErr != nil {
+			http.Error(w, "Error building page.", http.StatusInternalServerError)
+			return nil, wpErr
+		}
+		return nil, err
+	}
+
+	clones, err := client.GetAllClones(bug)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wpErr := writePage(w, "Not Found", errorTemplate, err.Error())
+		if wpErr != nil {
+			http.Error(w, "Error building page.", http.StatusInternalServerError)
+			return nil, wpErr
+		}
+		return nil, err
+	}
+	// Target versions would be used to populate the CreateClone dropdown
+	targetVersions := sets.NewString(allTargetVersions.List()...)
+	// Remove target versions of the original bug
+	for _, release := range bug.TargetRelease {
+		_, ok := targetVersions[release]
+		if ok {
+			targetVersions.Delete(release)
```

Comment:
You can just unconditionally do a `targetVersion.Delete`, it will work fine even if the `release` is not in the targetVersions (will just do nothing in that case).

Comment applies to all occurences of this.